src/processes/sd.json

```diff
@@ -0,0 +1,77 @@
+{
+  "id": "sd",
+  "summary": "Standard deviation",
+  "description": "Computes the sample standard deviation, which quantifies the amount of variation of an array of numbers. It is defined to be the square root of the corresponding variance (see ``variance()``).\n\nA low standard deviation indicates that the values tend to be close to the expected value, while a high standard deviation indicates that the values are spread out over a wider range.",
+  "categories": [
+    "math",
+    "reducer"
+  ],
+  "parameter_order": [
+    "data",
+    "ignore_nodata"
```

Comment:
ignore_nodata still in here without having a parameter, also examples need to be checked.